modules/cloudtrail/variables.tf

```diff
@@ -66,6 +66,12 @@ variable "event_selector" {
   default     = []
 }
 
+variable "insight_selector" {
```

Comment:
maybe better to have this in plural form as this is list?
so that the name if variable be "insight_selectors" instead "insight_selector"